scripts/train_linker.py

```diff
@@ -0,0 +1,72 @@
+from sklearn.ensemble import GradientBoostingClassifier
+from sklearn.metrics import accuracy_score, classification_report
+from joblib import dump, load
+import jsonlines
+import datetime
+from scispacy import umls_semantic_type_tree
+from linking import featurizer
+import argparse
+from tqdm import tqdm
+
+
+def main(data_path: str):
+    train = []
+    dev = []
+    test = []
+    root = ''
+    files = [f'{data_path}/train.jsonl', f'{data_path}/dev.jsonl', f'{data_path}/test.jsonl']
+    lists = [train, dev, test]
```

Comment:
Don't use `lists` as a variable name. It has no meaning. Maybe `data_splits`?

Reply:
ok